segmentstore/storage/impl/src/main/java/io/pravega/segmentstore/storage/impl/bookkeeper/DebugLogWrapper.java

```diff
@@ -243,6 +230,18 @@ public boolean reconcileLedgers(List<LedgerHandle> candidateLedgers) throws Dura
 
         return changed;
     }
+    
+    private static long getHandleLength(Handle handle) {
```

Comment:
it is because in theory this method can be called with a ReadHandle in a test, but actually it isn't
I feel it is safe to leave this form.
I will send a patch to BK in order to have getLength() available on Handle class

Reply:
I have double checked and WriteHandle is a subclass of ReadHandle. So code can be simpler

Reply:
Basically I have dropped this method